templates/workloads/services/lb-web/manifest.yml

```diff
@@ -9,9 +9,9 @@ type: {{.Type}}
 
 image:
```

Comment:
can we support the scheduled job manifest too?

Reply:
Not within this PR. I'll add job manifest support in a separate one!